samples/SnapshotConsoleSample/SnapshotConsoleSample.c

```diff
@@ -671,7 +784,8 @@ int main(int argc, char *argv[]) {
 			return 32;
 		}
 	} else {
-		if (!dxf_create_snapshot(connection, event_id, base_symbol, NULL, time_is_set ? time_value * 1000 : 0, &snapshot)) {
+		if (!dxf_create_snapshot(connection, event_id, base_symbol, NULL, from_time_is_set ? from_time_value * 1000 : 0,
```

Comment:
And probably we should add a check for non "indexed only" events here.
`Order, Spread Order, Series: from_time = 0`

Reply:
Done